xml/templates/message_templates/membership_online_receipt_text.tpl

```diff
@@ -170,8 +163,14 @@
 {$address}
 
 {$email}
-{/if} {* End ! is_pay_later condition. *}
-{/if}
+{if $contributeMode eq 'direct' AND !$is_pay_later AND ( $amount GT 0 OR $membership_amount GT 0 ) }
```

Comment:
This if can go

Reply:
Maybe after the other is merged because I forsee a conflict below

Reply:
@eileenmcnaughton it will in #15651